reflex/.templates/jinja/custom_components/pyproject.toml.jinja2

```diff
@@ -0,0 +1,35 @@
+[build-system]
```

Comment:
The general folder has limitation in certain ways using poetry. E.g., if we create new_component folder in reflex/examples and just the reflex in poetry environment, when we are in the new_component folder, it detects the pyproject.toml but no poetry section, it'll complain. Could just use `poetry shell` instead. Our user likely wont use it like this, so hopefully not interference with their choice of package manager.